### ISSUE TITLE
Absent lanugo

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7523]
**Preferred term label:**
Absent lanugo

**Synonyms**

**Definition (free text, please give PubMed ID)**
Lack of development of lanugo, the fine, soft, unpigmented hair on the body of a fetus or newborn baby. PMID: 30252348

 Comment:
Lanugo arises about three months into development and plays an essential role in binding the vernix caseosa to the skin of fetuses. 

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:200110 
PMID: 26119818
6/10

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
